config/spryk/spryks/Spryker/Glue/Processor/Mapper/AddGlueResourceMapperMethod.yml

```diff
@@ -0,0 +1,63 @@
+spryk: method
+description: "Adds a new resource mapper method to the Glue application."
+mode: both
+
+application: Glue
+
+priority: 1
+
+arguments:
+  module:
+    inherit: true
+
+  organization:
+    inherit: true
+    default: Spryker
+
+  resourceType:
+    inherit: true
+
+  fromTransfer:
+    inherit: true
+    default: "Rest{{ resourceType | dashToCamelCase }}AttributesTransfer"
+
+  toTransfer:
+    inherit: true
+    default: "Rest{{ resourceType | dashToCamelCase }}AttributesTransfer"
```

Comment:
Should this and `fromTransfer:` really be the same by default?

Reply:
This is probably fine if you plan this to be able to set only one of the values and the other will be set to this by default. I just think it's not too obvious when you look at the code, should be better in usage. 

No action here.